public/assets/css/author.css

```diff
@@ -0,0 +1,242 @@
+aside {
+    width: 25%;
+    padding-left: 15px;
+    font-family: Anisse;
+}
+
+aside>header div {
+    display: inline-block;
+    vertical-align: middle;
+    padding: 15px;
+    width: 130px;
+}
+
+aside>header img{
+    width: 25%;
+    height: 25%;
+    border-radius: 50%;
+    border: 5px solid rgb(0, 162, 255);
+    filter: drop-shadow(0 0 0.35rem rgb(61, 58, 58));
+    cursor: pointer;
+    display: inline-block;
+    vertical-align: middle;
+}
+
+aside>header img:hover {
+    filter: drop-shadow(0 0 0.35rem rgb(231, 227, 227));
+    box-shadow: 0 0 0.75rem rgb(0, 162, 255);
+}
+
+aside h1{
+    font-size: 12pt;
+    font-weight: 700; 
+}
+aside h2{
+    font-size: 9pt;
+}
+
+aside p {
+    font-size: 10pt;
+    padding-top: 30px;
+    padding-bottom: 20px;
+    text-align: justify;
+}
+
+.icon-rss {
+    border-radius: 50%;
+    background:  rgb(0, 162, 255);
+    opacity:0.5;
+    width: 60px;
+    height: 60px;
+    display: inline-block;
+    vertical-align: middle;
+    position: relative;
+}
+
+.icon-rss:hover {
+    opacity: 1;
+    -webkit-transform: scale(1.25);
+}
+
+.quart1 {
+    width: 33px;
+    height: 33px;
+    -webkit-border-radius: 0 0 100%;
+    border-radius: 0 0 100%;
+    background: white;
+    transform:rotate(-90deg);
+    position:relative;
+    top:10px;
+    left:17px;    
+}
+
+.quart2 {
+    width: 25px;
+    height: 25px;
+    -webkit-border-radius: 0 0 100%;
+    border-radius: 0 0 100%;
+    background: rgb(0, 162, 255);
+    transform: rotate(-90deg);
+    position: relative;
+    bottom: 15px;
+    left: 17px;    
+}
+
+.quart3 {
+    width: 17px;
+    height: 17px;
+    -webkit-border-radius: 0 0 100%;
+    border-radius: 0 0 100%;
+    background: white;
+    transform: rotate(-90deg);
+    position: relative;
+    bottom: 32px;
+    left: 17px;    
+}
+
+.quart4 {
+    width: 9px;
+    height: 9px;
+    -webkit-border-radius: 0 0 100%;
+    border-radius: 0 0 100%;
+    background: rgb(0, 162, 255);
+    transform: rotate(-90deg);
+    position: relative;
+    bottom: 40.5px;
+    left: 17px;    
+}
+
+.point {
+    width: 8px;
+    height: 8px;
+    border-radius: 50%;
+    background:white;
+    position: relative;
+    bottom: 46px;
+    left: 14px;
+}
+
+.icon-email {
```

Comment:
create a file path like this `css/icons/email.css` and  `css/icons/icons.css`

in `css/icons/icons.css` would contain a global rule of the articles and would import the other files, like this :

in file `css/icons/icons.css`
```css
@import url(./email.css)

.icon{
...
}
.icon:hover{
...
}
.icon-circle {
...
}
.icon-circle:hover{
...
}
```

in file `email.css`
```css
.icon .email{
...
}

.email-xxx{
...
}

/* override color for icon-circle */
.icon-circle .email-xxx{
...
}
```
or 
```css
.icon .icon-email{
...
}

.icon-email-xxx{
...
}

/* override color for icon-circle */
.icon-circle .icon-email-xxx{
...
}
```

Reply:
"xxx" rename like this exemple : "part-one", ...

```css
/* bem */
.icon-email-part--one{
...
}

```
or 
```css
/* oocss */
.icon-email-part-one{
...
}

```

Reply:
Email icon renamed accordingly